src/components/common/Tabs/QuestionTabs/QuestionTabs.tsx

```diff
@@ -1,40 +1,73 @@
 import { Box } from '@mui/material';
 import React, { useCallback, useRef, useState } from 'react';
-import { QuestionDataType } from '../../../../Mocs/QuestionTabsMoc';
-import { scrollToCenter } from '../../../../utils/scrollToCenter';
 import { Tabs } from '../commonStyles';
 import { QuestionTabsItem } from './QuestionTabsItem';
+import {
+  Difficulty,
+  TypeOptions,
+} from '../../../../types/CreateQuestionPropsType';
 
-type RectangleProgressBarPropsType = {
-  activeQuestionId: string;
-  questionsData: QuestionDataType[];
+type QuestionTabsType = {
+  numberOfQuestions: number;
+  activeTab: any;
 };
-export const QuestionTabs = (props: RectangleProgressBarPropsType) => {
-  const [activeTabId, setActiveTabId] = useState(
-    props.activeQuestionId || props.questionsData[0].questionId
-  );
 
+const getNewQuestion = () => ({
+  id: Math.random(), // ?
```

Comment:
just use counter, maybe in closure. Real id will be generated on BE after

Reply:
fix. i forgot to delete this field, couse i create id in another place